src/Document/BasePHPCRManager.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\Doctrine\Document;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Sonata\Doctrine\Model\BaseManager;
+
+/**
+ * @mixin ObjectManager
+ */
+abstract class BasePHPCRManager extends BaseManager
+{
+    /**
+     * Make sure the code is compatible with legacy code.
+     *
+     * @return mixed
+     */
+    public function __get($name)
+    {
+        if ('dm' === $name) {
+            return $this->getObjectManager();
+        }
+
+        throw new \RuntimeException(sprintf('The property %s does not exists', $name));
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
In this case, do we still remove `inheritdoc`?

Reply:
We shouldn't do this, cause the original documentation will be lost if you generate api docs

Reply:
Do people actually do this?

Reply:
That's how PHPDoc works...